### PR TITLE
🚀 General Availability CLI changes

### PR DESCRIPTION
**Pull request by gwyneplaine**
^ as it says on the tin. 
PR-ing this into the ga-release branch to make it easier to review the changes.

### CODE REVIEW

File: packages/keystone/README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# @keystone-6/keystone
+# @keystone-next/keystone
```
Comment (mitchellhamilton) [inline packages/keystone/README.md:1]: Was this intentional?

Comment (gwyneplaine) [reply packages/keystone/README.md:1]: yep 👍

Comment (gwyneplaine) [reply packages/keystone/README.md:1]: oh wait, nope! rebase problems

File: docs/pages/updates/index.tsx [hunk 2]
```diff
@@ -715,7 +715,7 @@ export default function WhatsNew() {
             <li>
               Exchanged <InlineCode>deploy</InlineCode>, <InlineCode>reset</InlineCode> and{' '}
               <InlineCode>generate</InlineCode> commands for{' '}
-              <InlineCode>keystone-next prisma</InlineCode> commands.
+              <InlineCode>keystone prisma</InlineCode> commands.
```
Comment (mitchellhamilton) [inline docs/pages/updates/index.tsx:718]: I would think we wouldn't want to update this(and the other bit in this file) given it's about a release and in that release, the command was `keystone-next`?

Comment (gwyneplaine) [reply docs/pages/updates/index.tsx:718]: ah good catch, missed this exclude in the find-replace

**Review by emmatown — APPROVED** [review]

**Pull request merged**